easybuild/easyconfigs/h/heaptrack/heaptrack-1.1.0-foss-2016b.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'CMakeMake'
+
+name = 'heaptrack'
+version = '1.1.0'
+
+homepage = 'http://milianw.de/blog/heaptrack-a-heap-memory-profiler-for-linux'
+description = """A heap memory profiler for Linux."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = ['https://github.com/KDE/heaptrack/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['bd247ac67d1ecf023ec7e2a2888764bfc03e2f8b24876928ca6aa0cdb3a07309']
+
+builddependencies = [
+    ('binutils', '2.26'),
+    ('CMake', '3.7.1')
+]
+
+dependencies = [
+    ('zlib', '1.2.8'),
+    ('zstd', '1.3.4'),
+    ('Boost', '1.63.0', '-Python-2.7.12'),
+    ('libunwind', '1.2.1'),
+    ('libdwarf', '20150310'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/heaptrack', 'bin/heaptrack_print', 'lib/heaptrack/libheaptrack_preload.so'],
```

Comment:
@vanzod Please use `SHLIB_EXT` rather than hardcoding `.so`

Reply:
Good catch. I forgot that...